frontend/src/components/Explore.tsx

```diff
@@ -49,16 +52,30 @@ export const Explore = (): JSX.Element => {
 		listDatasets(0, limit, mine);
 	}, []);
 
+	useEffect(() => {
+		listDatasets(0, limit, mine);
+	}, [mine]);
+
 	// Admin mode will fetch all datasets
 	useEffect(() => {
 		listDatasets(0, limit, mine);
 	}, [adminMode]);
 
+	// Admin mode will fetch my datasets
+	useEffect(() => {
+		listDatasets(0, limit, true);
+	}, [adminMode, mine]);
+
+
 	// switch tabs
 	const handleTabChange = (
 		_event: React.ChangeEvent<{}>,
 		newTabIndex: number
 	) => {
+		if (newTabIndex === 1) {
+			setMine(true);
+			listDatasets(0, limit, true);
```

Comment:
I don't think you need this `listDatasets`. You have a useEffect hook listen to Mine

Reply:
This is removed, not needed